scripts/BenchmarkGeoDB.java

```diff
@@ -0,0 +1,67 @@
+import com.maxmind.geoip2.DatabaseReader;
+import com.maxmind.geoip2.exception.AddressNotFoundException;
+import com.maxmind.geoip2.model.CityResponse;
+import java.io.File;
+import java.net.InetAddress;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class BenchmarkGeoDB {
```

Comment:
submit your benchmark stuff as a separate PR so it is easier for Philip to see.

Reply:
Removed from branch